UI/GardenDeskForm.cs

```diff
@@ -101,27 +92,293 @@ private async void BtnLogin_Click(object sender, EventArgs e)
+            comboBoxTypeUser.SelectedItem = selectedEmployee.Role.ToString();
+            textBoxEmailAddress.Text = selectedEmployee.Email;
+            textBoxPhoneNumber.Text = selectedEmployee.PhoneNumber;
+            textBoxBranch.Text = selectedEmployee.Branch;
+        }
+        #endregion
+        #region Delete employee logic
+        // method to delete the employee (my poor amigo)
+        private async void btnDeleteEmployee_Click(object sender, EventArgs e)
+        {
+            if (ConfirmDelete())
+            {
+                await employeeService.DeleteEmployeeByID(selectedEmployee.EmployeeId);
+                // confirm that the employee was deleted
+                MessageBox.Show($"{selectedEmployee.FirstName} {selectedEmployee.LastName} will be missed...");
+                ShowUsersView();
+            }
+        }
+        // ask the user to confirm deleting the employee
+        private bool ConfirmDelete()
+        {
+            // add a message box to get confirmation for deleting them
+            string messageTop = "Confirmation";
+            string messageText = "Are you sure you wish to delete this employee?";
+            return MessageBox.Show(messageText, messageTop, MessageBoxButtons.YesNo) == DialogResult.Yes;
+        }
+        #endregion
+        private void btnCancelChangesEmployee_Click(object sender, EventArgs e)
+        {
+            ShowUsersView();
+        }
+        #region Users view and buttons control
+        // method to show the panel, refresh the employees list and the buttons
+        private async void ShowUsersView()
         {
             ShowPanel(pnlUsers);
+            ChangeButtonState(btnEditEmployee, Color.LightGray, false);
+            ChangeButtonState(btnAddEmployee, Color.LightGreen, true);
             await DisplayEmployeesAsync();
         }
 
-        private async void menuItemIncedents_Click(object sender, EventArgs e)
+        // method to make the edit employee button enabled once the user has selected an employee
+        // and make the add user btn disabled
+        private void usersList_SelectedIndexChanged(object sender, EventArgs e)
         {
-            ShowPanel(pnlTicketsOverview);
-            await DisplayTicketsAsync(currentEmployee);
+            if (usersList.SelectedItems.Count > 0)
+            {
+                ChangeButtonState(btnEditEmployee, Color.Orange, true);
+                ChangeButtonState(btnAddEmployee, Color.LightGray, false);
+            }
+            // the add button becomes enabled once you have unselected
+            else
+            {
+                ChangeButtonState(btnEditEmployee, Color.LightGray, false);
+                ChangeButtonState(btnAddEmployee, Color.LightGreen, true);
+            }
+        }
 
+        // method to change the enablement and coloration of the buttons
+        // Do you want this method to be outside of this region if others' also need to change a button or something?
```

Comment:
Remove this comment. We can discuss things in the chat. Don't need to make a chat from the code.